lib/flame_slurm_backend/slurm_client.ex

```diff
@@ -60,10 +60,22 @@ defmodule FLAMESlurmBackend.SlurmClient do
     file = File.open!("flame_auto.sh", [:write])
 
     IO.puts(file, slurm_job)
+    # Ask SLURM to signal 30 sec before kill to send SIGUSR1 to shutdown BEAM
+    # Adds a folder in the TMPDIR and changes $TMPDIR to it
+    # to be able to delete it after the job
     IO.puts(file, """
-    elixir -e "$ELIXIR_SLURM_SCRIPT"
-    """)
-
+#SBATCH --signal=B:SIGUSR1@30
```

Comment:
SIGUSR1 will immediately shut it down and not wait for anything.... we may want to send `sigterm` instead with 30 seconds in advance and then a `sigusr1` with 5 seconds before?